docs/riff_streaming_inmemory-gateway_status.md

```diff
@@ -8,12 +8,12 @@ show inmemory gateway status
 
 ### Synopsis
 
-Display status details for a in-memory gateway.
+Display status details for a in-memory Gateway.
```

Comment:
a -> an

Reply:
Also, not sure all doc pages insist on using capitalized names for resources, _everywhere_.

Reply:
capitalization removed now

Reply:
and a->an now fixed